.travis.yml

```diff
@@ -21,4 +21,8 @@ script:
   - cabal build
   - cabal test
   - cabal install
+  - ([ "$GHCVER" = "7.6.3" ] || cabal configure --enable-tests --enable-executable-dynamic)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal build)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal test)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal install)
```

Comment:
yes, `build` is technically superfluous. The `test` invocation, I believe, isn't, because it builds a links a dynamic `unit-test` executable to use.